app/src/common/shared/org/mozilla/vrbrowser/telemetry/TelemetryWrapper.java

```diff
@@ -595,6 +595,26 @@ private static void queueActiveWindowPctEvent() {
         }
     }
 
+    public static void resetOpenedWindowsCount(int number, boolean isPrivate) {
+        if (isPrivate) {
+            for (int i=0; i<openPrivateWindows.length; i++) {
+                openPrivateWindows[i] = 0;
+            }
+            if (number > 0) {
+                openPrivateWindows[number -1 ] = 1;
```

Comment:
It makes sense to me for resetting to 0, but giving 1 looks odd to me

Reply:
The `openWindows` and `openPrivateWindows` have three buckets for 1, 2 or 3 opened windows during the application lifetime so we can calculate the weighted mean later. `resetOpenedWindowsCount` gets the current windows number as a parameter (as it happens upon onresume), so when resetting we set that bucket to 1 as that's the initial windows count.

Reply:
This also applies to above and below comments. `openWindows` has three buckets so when setting to one we are basically initializing the bucket for that windows number.